src/lib/queries/get-exists.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { HomeDocument } from 'test/module'
+import { setupNestApplication } from 'test/toolkit'
+import { TEST_ELASTICSEARCH_NODE } from 'test/constants'
+import { ElasticsearchModule } from 'module/elasticsearch.module'
+import { ElasticsearchService } from 'module/elasticsearch.service'
+import { getExistsQuery } from './get-exists'
+
+describe('getExistsQuery', () => {
+    const { app } = setupNestApplication({
+        imports: [
+            ElasticsearchModule.register({
+                node: TEST_ELASTICSEARCH_NODE
+            })
+        ]
+    })
+
+    it('accepts only schema fields', () => {
+        const query = getExistsQuery<HomeDocument>('address')
+
+        expect(query).toEqual({
+            exists: {
+                field: 'address'
+            }
+        })
+    })
+
+    it('queries es for range aggregation with nested aggregation', async () => {
+        const service = app.get(ElasticsearchService)
+        const result = await service.search(HomeDocument, {
+            size: 2,
+            query: {
+                bool: {
+                    must: getExistsQuery('propertyAreaSquared')
+                }
+            }
+        })
+
+        expect(result.documents).toEqual([
```

Comment:
I think it would be better if we don't use exact equal and instead use `expect.any(String)`

Reply:
the reason because, this test can render flaky if it won't return these exact two documents, I think we only care that it returns somewhat the structure of document - wdyt? 🙏

Reply:
it should check if documents does not ave null values for field I used in the query, so something in between should be ideal, but I got your point

Reply:
for reference https://jestjs.io/docs/expect#asymmetric-matchers